src/banman.h

```diff
@@ -68,6 +77,17 @@ class BanMan
     CClientUIInterface* m_client_interface = nullptr;
     CBanDB m_ban_db;
     const int64_t m_default_ban_time;
+
+    /** Individual addresses that of peers that misbehave (see Misbehaving() in
+     *  net_processing) are not banned but discouraged: we still allow incoming
+     *  connections from them, but prefer them for eviction. Similar to banned
+     *  IPs, we never make outgoing connections to them, and do not rumour them
```

Comment:
b6a34484 For my understanding, "rumour" == gossip?

Reply:
This is gone.